src/test/java/org/mifos/integrationtest/cucumber/stepdef/BatchApiStepDef.java

```diff
@@ -861,4 +861,59 @@ public void iCreateAListOfPayeeIdentifiersFromCsvFile() {
             logger.error(e.getMessage());
         }
     }
+
+    @When("I call the Batches API with invalid batchId expecting status of {int}")
+    public void iCallTheBatchesAPIWithInvalidBatchIdAndRequestidExpectingStatusOf(int expectedStatus) {
+        RequestSpecification requestSpec = Utils.getDefaultSpec(scenarioScopeState.tenant);
+        String batchId = "Jupiter";
+        // String requestId = "1234";
+
+        String endpoint = String.format("%s?batchId=%s", operationsAppConfig.batchDetailsEndpoint, batchId);
+        String fullUrl = operationsAppConfig.operationAppContactPoint + endpoint;
```

Comment:
the string. format in the above line can be used to create the full URL. no need to add this additional line

Reply:
fullUrl is not used anywhere other than logs